packages/core/src/browser/keybinding.ts

```diff
@@ -91,6 +91,30 @@ export namespace KeybindingContexts {
         id: 'default.keybinding.context',
         isEnabled: () => false
     };
+
+}
+
+/**
+ * Keybinding context that is enabled when the focused HTML element is an `input` or a `textArea`.
+ */
+@injectable()
+export class NativeTextInputFocusContext implements KeybindingContext {
+
+    static readonly ID = 'nativeTextInputFocus';
+    readonly id = NativeTextInputFocusContext.ID;
+
+    isEnabled(): boolean {
+        return NativeTextInputFocusContext.is();
+    }
+
+    /**
+     * `true` if the argument is an `input` or a `textArea`. Otherwise, `false`.
+     * The `element` argument defaults to the "focused" DOM element (`document.activeElement`).
+     */
+    static is(element: Element | null = document.activeElement): element is HTMLInputElement | HTMLTextAreaElement {
```

Comment:
I wonder why don't we inject `NativeTextInputFocusContext` and call `isEnabled`?